### ISSUE TITLE
LDAP support?

### ISSUE THREAD

**Issue by ghost** [mail-in-a-box/mailinabox#119]
Hi,

I use LDAP to provide a centralised password system.

How can I use mailinabox with LDAP?

Currently I have to add each user manually with `./tools/mail.py add new_user` - password management is becoming an issue...

Is an LDAP tweak hard to do?

Fantastic project by the way!

**Comment by arodier** [comment]
Hello Josh et al,

If this can help, I have started to publish and improved the scripts I used to deploy my personal mail server at home. I always used LDAP for SSO.

The whole installation is automated through Ansible, but the LDAP part is easy:
https://github.com/progmaticltd/homebox/blob/dev-arodier/install/playbooks/roles/ldap/tasks/main.yml

Perhaps I will add a reference to your project in my readme file, this is a fantastic work.

Any question welcome.
